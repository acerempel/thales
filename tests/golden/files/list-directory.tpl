<article>
  <h1>Contents of directory:</h1>
  <p>
    <ul>
      {for file in list-directory ("list-directory-files")}
      <li>{file}</li>
      {end}
    </ul>
  </p>
</article>
