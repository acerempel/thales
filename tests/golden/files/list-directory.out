<article>
  <h1>Contents of directory:</h1>
  <p>
    <ul>
      
      <li>a.txt</li>
      
      <li>b.txt</li>
      
      <li>c.txt</li>
      
    </ul>
  </p>
</article>
